config/crd/bases/activemonitor.keikoproj.io_healthchecks.yaml

```diff
@@ -3,8 +3,6 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  annotations:
-    controller-gen.kubebuilder.io/version: v0.3.0
```

Comment:
is this an generated code that removed it?

Reply:
reverted these changes.